usagov/vote-gov
# Vote 789 -> update language switcher test

link to ticket -> https://bixal-projects.atlassian.net/browse/VOTE-789 and https://bixal-projects.atlassian.net/browse/VOTE-788

This PR will cover 2 tickets... this will update the language switcher test to reflect the new button format and will also add the new Navajo option to the test.

This has been updated to run through a list of URLs and then check that the vote.gov logo is correct. I did this to add the home page so that this one function is covered by this test because the addition of the homepage is why I am now targeting the logo text since it will all be the same on each page.  This will also help with the move to making the tests more component based and not just per page or how we were doing it. 

I have also removed the language switcher test from the home page and will revisit that one at a later date.

To test:

- pull down the branch 
- run `npm run start` 
- spilt terminal 
- run `npm run cy:open`
- select the `language-switcher.cy.js` file and verify that the test is passing.
